### DISCUSSION TITLE
About Assignment of HouseWare .

### DISCUSSION THREAD

**Discussion by lakshay1121** [community/community#51146] [GitHub Education] [answered]
I have completed the Houseware assignment and made the changes to the auto generated feedback branch and also raised the PR for reviewing but I did not got any reply from anyone yet or any notification regarding the assignment? . How much time I should have to wait or if anyone do know how to contact the houseware team regarding the octernship.

**Comment by CodeMaster7000** [comment, marked as answer]
Hi @lakshay1121,

How long the Houseware team takes to respond depends on how busy they are with PRs at the moment. If you wish to contact the Houseware team, check if discussions are open in the repository. If not, I would recommend opening an issue to alert attention.

Thank you for asking and have a great day!

**Reply by lakshay1121** [reply]
Thanks alot!
